infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -345,7 +345,7 @@ public TopologyDescriptor generateConfigWithBackup() {
         newActiveClusters.get(0).getNodesDescriptors().add(backupNode);
         List<ClusterDescriptor> standbyClusters = new ArrayList<>(currentConfig.getStandbyClusters().values());
 
-        return new TopologyDescriptor(++configId, newActiveClusters, standbyClusters);
+        return new TopologyDescriptor(configId, newActiveClusters, standbyClusters);
```

Comment:
For future reference, can we add a comment here indicating why we are no longer bumping up the version, basically because this is an observed behavior on prod environment. Also, it might be good idea to have two flavors of the backup restore, one where the version is bumped and another where it isn't. Since we do not control what the topology provider is doing, it should work seamlessly for either.